pages/content/amp-dev/documentation/guides-and-tutorials/optimize-measure/publishing_checklist.md

```diff
@@ -0,0 +1,121 @@
+
+Learn more about the tools you can use to validate your AMP pages:
+
+
+
+*   [Validate AMP pages]({{g.doc('/content/amp-dev/documentation/guides-and-tutorials/learn/validation-workflow/validate-amp.html', locale=doc.locale).url.path}}?format=websites)
+*   [The AMP Validator ](https://validator.ampproject.org/)
+*   [Google AMP Tester](https://search.google.com/test/amp)
+*   [AMP Tools]({{g.doc('/content/amp-dev/documentation/documentation/tools.html', locale=doc.locale).url.path}}?format=websites) 
+
+
+# Grant cached AMP documents server access
+
+Valid AMP pages are automatically opted into all existing AMP caches. This is great news! Your pages are automatically preloaded efficiently and safely, enabling user-beneficial performance optimizations to content. However, documents served from AMP caches are served from domains that do not match your own. This can cause  Cross-Origin Resource Sharing (CORS) issues when using dynamic AMP components, such as [`<amp-form>`]({{g.doc('/content/amp-dev/documentation/components/reference/amp-form.html', locale=doc.locale).url.path}}?format=websites) or [`<amp-list>`]({{g.doc('/content/amp-dev/components/reference/amp-list.html', locale=doc.locale).url.path}}?format=websites), causing your site to lose access to your data. This means certain content may not load correctly or at all.  It's important to enable CORS Requests from all available [AMP Caches](https://cdn.ampproject.org/caches.json) for your pages to work as intended for all possible users. 
+
+Learn more about accessing server data:
+
+
+
+*   [How AMP Pages are cached ]({{g.doc('/content/amp-dev/documentation/guides-and-tutorials/learn/amp-caches-and-cors/how-cached.html', locale=doc.locale).url.path}}?format=websites)
+*   [CORS in AMP]({{g.doc('/content/amp-dev/documentation/guides-and-tutorials/learn/amp-caches-and-cors/amp-cors-requests.html', locale=doc.locale).url.path}}?format=websites)
+*   [AMP CORS Library](https://www.npmjs.com/package/amp-toolbox-cors)
+
+
+#Test pages in the Google AMP Cache
+The Google AMP Cache stores valid AMP pages and provides consistently fast access to them. The [Google Top Stories Carousel with AMP](https://developers.google.com/search/docs/guides/mark-up-content#use-amp-html), for example, uses the Google AMP Cache to display articles. The cache stores images and fonts in addition to documents. This makes it important to test that your AMPs work correctly when loaded via the Google AMP Cache.
+
+{{ image('/static/img/docs/amp_error.png', 385, 386, layout='intrinsic', alt='Image showing an error from the Google AMP Cache') }}
+
+Loading your AMP pages via the Google AMP Cache is easy. The Google AMP Cache URL is composed based on whether the source URL is available via HTTP or HTTPS:
```

Comment:
I think the whole section is a bit redundant to 

https://github.com/ampproject/docs/blob/future/pages/content/amp-dev/documentation/guides-and-tutorials/learn/amp-caches-and-cors/amp-cache-debugging.md

and

https://github.com/ampproject/docs/blob/future/pages/content/amp-dev/documentation/guides-and-tutorials/learn/amp-caches-and-cors/amp-cache-urls.md

I'd instead link to these/